include/aspect/material_model/rheology/strain_dependent.h

```diff
@@ -98,6 +98,16 @@ namespace aspect
           compute_strain_weakening_factors(const unsigned int j,
                                            const std::vector<double> &composition) const;
 
+          /**
+           * bla
+           */
+
+          std::array<double, 3>
+          apply_temperature_dependence_to_strain_weakening_factors(const unsigned int j,
+                                                                   const double temperature,
+                                                                   std::array<double, 3> weakening_factors) const;
```

Comment:
I know that `compute_strain_weakening_factors` also puts `j` as first parameter, but I dont think that is very intuitive. I would follow the convention of the other functions in strain_dependent.h and put `j` last. I think the most intuitive ordering would be:
```
apply_temperature_dependence_to_strain_weakening_factors( ... weakening_factors, ... temperature, ... j)
```